Get Assets - Oxmaint/readme.md

```diff
@@ -0,0 +1,31 @@
+## Get Assets Oxmaint

```

Comment:
Please update readme.md file with all the required sections mentioned in the below url. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md

Reply:
updated the readme.md file